packages/google_mlkit_barcode_scanning/README.md

```diff
@@ -58,16 +58,16 @@ final List<Barcode> barcodes = await barcodeScanner.processImage(inputImage);
 for (Barcode barcode in barcodes) {
   final BarcodeType type = barcode.type;
   final Rect boundingBox = barcode.boundingBox;
-  final String displayValue = barcode.displayValue;
-  final String rawValue = barcode.rawValue;
+  final String? displayValue = barcode.displayValue;
+  final String? rawValue = barcode.rawValue;
 
   // See API reference for complete list of supported types
   switch (type) {
     case BarcodeType.wifi:
-      BarcodeWifi barcodeWifi = barcode.value;
+      BarcodeValue? barcodeWifi = barcode.value;
```

Comment:
change to 
`final barcodeWifi = barcode.value as BarcodeWifi;`

Reply:
okay